Archives/ArchivePacker.cpp

```diff
@@ -46,7 +51,11 @@ namespace Archives
 	bool ArchivePacker::ReplaceFileWithFile(const char *fileToReplace, const char *newFile)
 	{
 		MoveFileA(fileToReplace, "Delete.vol");
```

Comment:
When I implemented ADD and REMOVE using OP2Archive, I didn't use ArchivePacker::Repack. Instead I followed the following workflow:

1. Create a temp directory
2. extract contents of vol file to temp directory
3. modify temp directory files by either adding files from other sources or removing files
4. Delete original volume.
5. Pack new volume with original name.

I would be fine removing Repack and this function altogether from the library. 

Here is the function to create a random directory:

```
string ConsoleHelper::CreateTempDirectory()
{
	srand(static_cast<int>(time(NULL))); // For creating a unique directory.
	int number = rand();

	string directory("./OP2ArchiveTemp-" + to_string(number));

	XFile::NewDirectory(directory);

	return directory;
}
```

Reply:
Perhaps we should revisit this at a future time then. Which makes sense, since that part wasn't even changed in your commits, just something I noticed while reviewing.